### ISSUE TITLE
Repurpose lesson to regular expressions, move intro, jargon, foundations to overview

### ISSUE THREAD

**Issue by libcce** [LibraryCarpentry/lc-data-intro#146]
- Intro to LC and Foundations are used optionally by instructors per feedback collected by the maintainers
- [ ] Review Workshop Overview to A) see if using the overview is how we want to proceed moving forward B) if there are errors/edits that need to be fixed
- [ ]  Intro to LC has been revamped, is now optional, and moved to Workshop Overview:
https://github.com/LibraryCarpentry/lc-overview/blob/gh-pages/_episodes/02-intro-to-library-carpentry.md --> Recommend deleting Intro to LC in Intro to Data
- [ ] Foundations has been revamped, is now optional, and moved to Workshop Overview:
See https://github.com/LibraryCarpentry/lc-overview/tree/gh-pages/_episodes and Keyboard Shortcuts, File Naming, Computational Approach, Further Reading episodes --> Recommend deleting Foundations in Intro to Data
- [ ] Jargon Busting has been moved to Workshop Overview:
https://github.com/LibraryCarpentry/lc-overview/blob/gh-pages/_episodes/03-jargon-busting.md --> Recommend deleting Jargon Busting from Intro to Data
- [ ] Recommend splitting the Regular Expressions episode into two episodes, the new episode will start at Exercise Using Regex101.com
- [ ] Recommend renaming the lesson to Regular Expressions

**Comment by libcce** [comment]
@antonangelo you are still not listed as a Maintainer (in the team site) for this lesson. Can you check to see if you got a message from GitHub asking you to join the lesson team?

**Comment by ccronje** [comment]
Thanks for all this hard work @libcce! I've reviewed the Workshop Overview and agree with the list above. Thanks for updating the lesson listing also.

**Comment by libcce** [comment]
There was one thing that I couldn't figure out, why the link to the intro to LC was not working. We may need to reorder the episodes as well?

**Comment by libcce** [comment]
We will also have to figure who the maintainers of the overview will be. Do some of split off to maintain the regex lesson and some of us split off to maintain the overview? For now, I just moved the intro to data maintainers to overview.

**Comment by ccronje** [comment]
@libcce https://github.com/LibraryCarpentry/lc-overview/blob/gh-pages/_episodes/02-intro-to-library-carpentry.md doesn't have a header section - that may be the problem?

Do you have a particular concern about the order of the episodes? If yes, let's ping lc-data-intro-maintainers.

Good idea moving the lc-data-intro-maintainers to the overview. Let's check with the group on teams for volunteers for the regex lesson. Is it time to put a call out to the LC community to invite new maintainers and/or relieve current folks?

**Comment by libcce** [comment]
@ccronje it’s funny I just added a header. Looks like it’s working now. 

I did wonder if the episode order was fine so having additional feedback would be good.

Would be great to hear from the others about how we manage the Maintainers on the two lessons now.

**Comment by ccronje** [comment]
@libcce excellent :-) I'll start a discussion thread in lc-data-intro-maintainers

**Comment by ccronje** [comment]
Tasks left to do on this issue:
[ ] Delete Intro to LC from Intro to Data
[ ] Delete Foundations from Intro to Data
[ ] Delete Jargon Busting from Intro to Data
[ ] Split Regular Expressions episode in Intro to Data into two episodes (new episode will start at Exercise Using Regex101.com)
[ ] Update episode numbers and headers in Intro to Data
[ ] Rename Intro to Data lesson to Regular Expressions
[ ] Update links impacted by lesson name change (will need to check lessons' setup pages, README files, LC website) << lesson name changes have been problematic in the past, is there a checklist?

**Comment by libcce** [comment]
Removed intro to LC https://github.com/LibraryCarpentry/lc-data-intro/pull/149

**Comment by libcce** [comment]
Remove foundations https://github.com/LibraryCarpentry/lc-data-intro/pull/150

**Comment by libcce** [comment]
Remove jargon busting https://github.com/LibraryCarpentry/lc-data-intro/pull/151

**Comment by libcce** [comment]
Hi @ccronje @sharilaster @ppival @antonangelo all the changes have been addressed but for the moment I've kept the current repo name so that we don't have some of the linking issues pointed out by @ccronje. One other thing we forgot was updating the extra pages which I've done. Also, I've created a callout on the lesson overview page which talks about how regex is taught. If you like what you see, thumbs up this comment and we can close the issue. I do think there is still work to be done on the lesson content and the times may need to be adjusted but these can be issues/PRs as time goes by, as we catch things.
